game_engine.properties

```diff
@@ -1,3 +1,4 @@
+engine_version = 1.8.0.10
```

Comment:
new engine_version property being added here - this is the one location where we will have this value!